spec/my_inject_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative '../lib/my_enumerables'
+
+RSpec.describe Enumerable do
+  subject(:enumerable) { [1, 1, 2, 3, 5, 8, 13, 21, 34] }
+
+  describe '#my_inject' do
```

Comment:
Should we include a test case to ensure it uses the initial value for the first iteration

Maybe something like this:
```
it 'uses the memo as the first value' do
  memo = 10

  expect(enumerable.my_inject(memo) { |sum, value| sum + value }).to eq(98)
end
```